app/views/user_mailer/account_activation.text.erb

```diff
@@ -0,0 +1,5 @@
+Hi <%= @user.name %>,
+
+Welcome to the Sample App! Click on the link below to activate your account:
```

Comment:
em tìm hiểu thêm về I18n để sửa lại đoạn này nhé. Search với từ khóa I18n in Rails